src/js/inPageNav.js

```diff
@@ -4,54 +4,160 @@ class InPageNav {
 
 	constructor (InPageNavEl, opts) {
 		this.inPageNavEl = InPageNavEl;
-		this.opts = opts || {}; // no opts right now
+		this.opts = opts || InPageNav.getOptions(InPageNavEl);
+
+		// the thing to look for within the container. Defaults to h2
+		if (this.opts.headingsSelector) {
+			this.headingsSelector = this.opts.headingsSelector;
+		} else {
+			 this.headingsSelector = 'h2';
+		}
+
+		// the container of the menu content. Defaults to body.
+		if (this.opts.containerEl) {
```

Comment:
I think the statements in this are the wrong way round? If there's an `opts.containerEl` it should set that to the container.